specifications/api/swagger.yaml

```diff
@@ -1409,12 +1416,37 @@ components:
           type: array
           items: *ApiWalletDelegationNext
 
+    AnyAddress: &AnyAddress
+      type: object
+      required:
+        - address
+      properties:
+        address: *anyAddress
+
+    ApiSharedScript: &ApiSharedScript
+      description: Discovered script that contains our keys
+      type: object
+      required:
+        - script_address
+        - our_keys
+        - funds_status
+        - funds_available
+      properties:
+        script_address: *AnyAddress
```

Comment:
Seems like there's some excessive wrapping here. The resulting JSON will look like `{ "script_address": { "address": "..." } }`

What about simply showing it as `"address": ...` ?